04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,79 @@
+const Product = require("../models/product");
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .sort("price")
+    .select("name price");
+  res.status(200).json({ products, nbHits: products.length });
+};
+
+const getAllProducts = async (req, res) => {
+  const { featured, company, name, sort, fields, numericFilters } = req.query;
+  const queryObject = {};
+
+  if (featured) {
+    queryObject.featured = featured === "true" ? true : false;
```

Comment:
nit: I know this is how it was done in the video, but just so you know, there's never a reason to have true and false like this in a ternary. It can just be `featured === 'true'`